Makefile

```diff
@@ -130,4 +130,4 @@ release: tar
 	@make clean-tar
 
 vendor-ansible:
-	git subtree pull --prefix ansible https://github.com/contiv/ansible HEAD --squash
```

Comment:
`--squash` causes a lot of problems. it can't trace back the commits that generated the squash, so the next pull breaks.

At least, this is what I am seeing in volplugin. Note https://github.com/erikh/volplugin/commit/8b2674fdb7bd8bb9dda3c2417d1bb32fbbe13bc8 which had to be reverted because of this.